plugins/outputs/nats/nats.go

```diff
@@ -85,7 +100,47 @@ func (n *NATS) Connect() error {
 
 	// try and connect
 	n.conn, err = nats.Connect(strings.Join(n.Servers, ","), opts...)
```

Comment:
We should check and return the error here before going on right?

Reply:
Done